install_scripts/templates/swarm/worker-join.sh

```diff
@@ -152,6 +152,7 @@ fi
 promptForSwarmMasterAddress
 
 if [ -n "$PROXY_ADDRESS" ]; then
+    # TODO: add private address cidr range to no proxy
```

Comment:
this is tough because all i have is the master address. i need to match the network that the master address is in from the ip command